### ISSUE TITLE
Direction Matters: On the Implicit Regularization Effect of Stochastic Gradient Descent with Moderate Learning Rate

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#58]
> この場合、SGDとGDは一意の最小ノルム解に収束することが知られているが、中程度の学習率とアニーリング学習率では、SGDはデータ行列の大きな固有値方向に沿って収束するのに対し、GDは小さな固有値方向に沿って収束するという異なる方向性のバイアスを示す。

https://arxiv.org/abs/2011.02538

UCLA
Johns Hopkins University

<img width="575" alt="Screen Shot 2020-11-22 at 10 25 33" src="https://user-images.githubusercontent.com/8721858/99891457-30863c00-2cad-11eb-81f1-7d20e1542e2e.png">
<img width="572" alt="Screen Shot 2020-11-22 at 10 25 45" src="https://user-images.githubusercontent.com/8721858/99891460-3419c300-2cad-11eb-9781-c7759cca9d89.png">
